saplings/circuits/src/App.js

```diff
@@ -20,13 +20,18 @@ import { faPlus } from '@fortawesome/free-solid-svg-icons';
 import { library } from '@fortawesome/fontawesome-svg-core';
 
 import MainHeader from './components/MainHeader';
+import { LocalNodeProvider } from './state/localNode';
+import Content from './components/Content';
 
 library.add(faPlus);
 
 function App() {
   return (
     <div className="circuits-app">
-      <MainHeader />
+      <LocalNodeProvider>
```

Comment:
I think the LocalNodeProvider could be the top element here (Around div.circuits-app)

Reply:
I don't know if that is necessary. The idea is that the `circuits-app` div is wrapping everything in the circuits sapling, that includes the LocalNodeProvider